src/main/resources/templates/add.ftl

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<body>
+<h3>Adding new blog</h3>
```

Comment:
The main header of any page usually is in "h1" tag, if you don't like the size of the text, just decrease it with "font-size" property, for example in an external css file